lib/ae_mdw/oracles.ex

```diff
@@ -332,6 +333,8 @@ defmodule AeMdw.Oracles do
       active: is_active?,
       active_from: register_height,
       expire_height: expire_height,
+      approximate_expiration_time:
```

Comment:
Good call on leaving `expire` instead of `expiration` to follow `expire_height` format. Left the `approximate` because it works well for both past and future dates